howto/manage-models.md

```diff
@@ -1,6 +1,95 @@
 (manage-models)=
 # Manage models
 
+(creating-a-model)=
+## Creating a model
+
+Creating a model with JAAS is similar to creating one with Juju.
+
+This section will cover:
+- How permissions for model creation are defined.
+- How controller selection is performed.
+- How to specify a specific controller to host your model.
+
+### Add-model permissions
+
+Permissions to add a model are based on a user's access to the desired
+cloud and controllers.
+
+Creating a model requires access to two things:
+- The `can_addmodel` permission on the target cloud.
+- The `can_addmodel` permission on one or more controllers that support that cloud.
+
+By default, all users have add-model access to all clouds in JAAS.
+Inversely, users are granted no default access to any controllers.
+
+See our {doc}`permission management doc <./manage-permissions>` for more details.
+
+### Controller selection
+
+Keeping in mind that JAAS is a manager of multiple controllers,
+the Juju `add-model` command will create a model on any controller
+your user can access that supports the specified cloud. JAAS will 
+prioritise controllers within that cloud to reduce latency and select
+randomly when there are multiple valid options.
+
+Consider the example below:
+
+
+```text
+juju add-model openstack my-model
+```
+
+```{mermaid}
+flowchart LR
+    U["User </br> (limited controller access)"]
+
+    subgraph Controllers
+        C1["Controller A</br>(supports openstack)"]
+        C2["Controller B</br>(supports openstack)"]
+        C3["Controller C</br>(does NOT support openstack)"]
+        C4["Controller D</br>(supports openstack)"]
+    end
+
+    U -- access --> C2
+    U -. no access .-> C1
+    U -. no access .-> C4
+    U -. no access .-> C3
+
+    classDef ok fill:#b3e6b3,stroke:#2d662d,stroke-width:1px;
+    classDef no fill:#f2b3b3,stroke:#662d2d,stroke-width:1px;
+
+    class C1,C2,C4 ok;
+    class C3 no;
+
+    %% Model placement result
+    subgraph Result
+        M[(my-model)]
+    end
+
+    C2 -- selected for model --> M
+```
+In the diagram above, multiple controllers shown in green support the `openstack`
+cloud while controller C (in red) does not. The dotted lines show that user does
+not have access to any controller except controller B.
+Based on these 2 factors the only valid placement is controller B.
+
+### Specifying a target-controller
+
+To specify the target controller that will host a model, there are 2 suggested
+approaches.
+1. Limit the controllers a user can access.
```

Comment:
Rather than say following option 1, I think just explain the entire approach in 1 and 2. because just saying "use the jaas plugin" doesn't really mean anything until I read below anyway.